models/README.md

```diff
@@ -0,0 +1,16 @@
+# Models
+
+Please refer [training]() section to understand how our best model is trained
+
+There are **three** main models in this repository
+
+1. Lexical Model - Uses lexical features like POS tags, Named entities etc., followed by a Conditional Random Field model
+2. Lexical Plus Model - In addition to the lexical features, we engineered more optimization specific features followed by a Conditional Random Field model
+3. (Our best) **Hybrid Model** - In addition to lexical features, engineered features, we added transformer predictions as a feature followed by a Conditional Random Field model
+
+## Directory structure
```

Comment:
the requirements.txt libraries are able to run all these pickles, correct?

Reply:
If I understand correctly, your question is why we need a bash script right, instead of using requirements.txt file? So, I wasn't able to find an allennlp version, which keeps my torch and spacy version as I wanted. We don't need allennlp for the library, it is a dependency from original NL4Opt Utils. What I did now, which I checked on Ubuntu works, is, install allennlp, overrides first (dependencies for NL4Opt) and then install Ner4Opt. Also, added the updated log.